sim/core/debuffs.go

```diff
@@ -67,7 +67,8 @@ func applyDebuffEffects(target *Unit, targetIdx int, debuffs *proto.Debuffs, rai
 	}
 
 	if debuffs.ImprovedScorch && targetIdx == 0 {
-		MakePermanent(ImprovedScorchAura(target))
+		aura := ImprovedScorchAura(target, 5)
+		MakePermanent(aura)
```

Comment:
Scorch should not start at 5 stacks even in the external version. You can check the Sunder Armor application and copy it for here.